data/test/scenarios/wml_tests/UnitsWML/zero_experience.cfg

```diff
@@ -0,0 +1,80 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [unit] max_experience=0
+##
+# Actions:
+# Change Alice's and Bob's max_experience to 0. Delete Bob's 'Orcish Warrior' advancement but not Alice's. Harm bob with 1 hitpoint to be sure that he have some damage. They then fight each other.
+##
+# Expected end state:
+# Alice levels up. Bob doesn't, and keeps his previous health.
+#####
+{GENERIC_UNIT_TEST "zero_experience" (
```

Comment:
There's a `COMMON_KEEP_A_B_UNIT_TEST` macro which means Alice and Bob already start adjacent to each other.

Reply:
I didn't know there were different maps for each test. I'll keep it in mind.